templates/_linux.yaml

```diff
@@ -50,6 +53,24 @@ objects:
       vm.kubevirt.io/template.revision: "{{ lookup('env', 'REVISION') | default(1, true) }}"
       app: ${NAME}
   spec:
+    dataVolumeTemplates:
+      - apiVersion: cdi.kubevirt.io/v1alpha1
+        kind: DataVolume
+        metadata:
+          name: ${PVCNAME}
```

Comment:
This looks problematic, if 2 or more VMs from the same template were to be generated, then a name collision would occur and the newer DV would fail to create or override the older one (depends on the implementation in kubevirt/kubevirt).
I think you should reuse `${NAME}` here, to guarantee that the DV name is generated and matches the VM's name